tests/CMakeLists.txt

```diff
@@ -339,7 +339,10 @@ else()
   set(DSLASH_POLICIES -1)
 endif()
 
+set(SPLIT_GRID_POLICIES 0 1) # whether or not split the grid
```

Comment:
This reminds me I really need to get the unit tests running faster since I thought before this they take way too long..... 😜